packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -401,11 +401,8 @@ module.exports = function (log, config, bounces) {
   Mailer.prototype._constructLocalTimeString = function (
     timeZone,
     acceptLanguage,
-    date,
-    time
   ) {
-    const translator = this.translator(acceptLanguage);
```

Comment:
Removing this line allowed the date and time to be converted to another language 🤔